crates/cranelift/src/func_environ.rs

```diff
@@ -1033,9 +1077,16 @@ impl<'module_environment> cranelift_wasm::FuncEnvironment for FuncEnvironment<'m
             WasmType::FuncRef => match plan.style {
                 TableStyle::CallerChecksSignature => {
                     let table_entry_addr = builder.ins().table_addr(pointer_type, table, index, 0);
-                    builder
-                        .ins()
-                        .store(ir::MemFlags::trusted(), value, table_entry_addr, 0);
+                    // Set the "initialized bit". See doc-comment on
+                    // `REF_INIT_BIT` in crates/runtime/src/table.rs
+                    // for details.
```

Comment:
Reading this again, could the constants here be extracted to something in the `wasmtime_environ` crate to be shared between compile-time and runtime?

Reply:
Done!